### PR TITLE
Fix format of percentage numbers

### PR DESCRIPTION
**Pull request by KyleMacPherson**
The percentage figures for metrics are not presented as percentages but as whole numbers on the single page view.
This PR fixes that.

https://trello.com/c/DhJHJL6b/679-1-fix-formatting-of-numbers-percentages-for-single-page

### CODE REVIEW

**Comment by KyleMacPherson** [comment]
[(1) Fix formatting of numbers/percentages (for single page)](https://trello.com/c/DhJHJL6b/679-1-fix-formatting-of-numbers-percentages-for-single-page)

File: app/presenters/chart_presenter.rb [hunk 1]
```diff
@@ -42,6 +42,8 @@ def keys
   end
 
   def values
-    json[metric].map { |hash| hash['value'] }
+    json[metric].map do |hash|
+      metric == :satisfaction_score ? (hash['value'] * 100).round : hash['value']
```
Comment (pmanrubia) [inline app/presenters/chart_presenter.rb]: For the time being can we extract a helper to render the satisfaction score?

```ruby
json[metric].map { |hash| MetricsFormatterHelper.format(metric, hash['value']) }
```

We should be able to use it in other places as well.

File: app/presenters/single_content_item_presenter.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ def parse_metrics(metrics)
     @unique_pageviews = metrics[:unique_pageviews]
     @pageviews = metrics[:pageviews]
     @number_of_internal_searches = metrics[:number_of_internal_searches]
-    @satisfaction_score = metrics[:satisfaction_score]
+    @satisfaction_score = 100 * metrics[:satisfaction_score]
```
Comment (pmanrubia) [inline app/presenters/single_content_item_presenter.rb]: I would rely on the helper. See comment above.

File: app/views/components/_info-metric.html.erb [hunk 3]
```diff
@@ -7,11 +7,12 @@
   context ||= false
 
 %>
+<% percentage_metrics = ['User satisfaction score'] %>
 <% if name && figure && context %>
   <div class="app-c-info-metric govuk-body">
     <h3 class="app-c-info-metric__heading"><%= name %></h3>
     <p class="app-c-info-metric__context govuk-body-s"><%= context %></p>
-    <span class="app-c-info-metric__figure govuk-!-font-weight-bold govuk-!-font-size-36 "><%= figure %></span>
+    <span class="app-c-info-metric__figure govuk-!-font-weight-bold govuk-!-font-size-36 "><%= figure %><%= '%' if percentage_metrics.include? name %></span>
```
Comment (pmanrubia) [inline app/views/components/_info-metric.html.erb]: I am not clear on why we need this. It doesn't feel totally right.

File: app/presenters/chart_presenter.rb (hunk 1 above)
Comment (pmanrubia) [reply app/presenters/chart_presenter.rb]: Also the helper can rely on Rails helper to render numbers / percentages.

**Pull request merged**